### DISCUSSION TITLE
Matrix profile as statistical distance

### DISCUSSION THREAD

**Discussion by ChrisCalvin** [TDAmeritrade/stumpy#963] [Q&A] [answered]
Can a matrix profile be used to compare the distance between two probability distributions?  I’m wondering if the MP can be used in a Bayesian model to replace a KL Divergence.

**Comment by seanlaw** [comment, marked as answer]
@ChrisCalvin Thank you for your question. At the end of the day, you'd essentially be computing the (z-normalized) Euclidean distance between the two distributions. I'm not sure what that would mean and you don't really need STUMPY for that as you likely wouldn't benefit from the sliding window component.
